_docker/README.md

```diff
@@ -0,0 +1,36 @@
+# Red Hat Developers Site Docker section
+
+Link back above [Link here](../README.md)
+
+## First steps
+Please follow the docker set up instructions in the main [README file](../README.md)
+
+## Running the unit tests
+The unit tests are fast to run, hence they get run for most tasks via the control script. To run them in isolation run
+
+```bash
+bundle exec ./control.rb -t
+```
+
+## Working on content / page design
+When working on site content or design, it's probably best to have the site running in live-reload mode. This mode is slow, and slightly unreliable with regard acceptance tests. However it is probably the most productive way to work as it does not require a new build each time. To do this run:
+
+```bash
+bundle exec ./control.rb --run-the-stack
+```
+
+This will allow you to work with any of the pages generated by awestruct. If you want to quit, then `CTRL-C` to stop.
+
+## Running the acceptance tests
+Every commit will be acceptance tested once a Pull Request is open. To run this via docker then the command is :
+
+```bash
+bundle exec ./control.rb --acceptance_test_docker
```

Comment:
When I run this I see the following when accessing: http://docker:32768/

      Internal Server Error

      undefined method `[]' for nil:NilClass
      WEBrick/1.3.1 (Ruby/2.1.2/2014-05-08) at docker:32768

Want to take a look when you get in the office tomorrow?